cg/cli/compress/fastq.py

```diff
@@ -71,7 +71,7 @@ def fastq_cmd(
 @click.option(
     "-b",
     "--days-back",
-    default=60,
+    default=21,
```

Comment:
I think this should be kept at 60. Given that it is days since the family was created (ie order placed) having it as 21 risks starting compression for cases where FASTQ files are still relevant. I would prefer a change in the query for cases to compress, and then change the days to 21

Reply:
Good point. Reverted